pyproject.toml

```diff
@@ -34,3 +34,5 @@ build-backend = "poetry.core.masonry.api"
 yoeo-detect = "yoeo.detect:run"
 yoeo-train = "yoeo.train:run"
 yoeo-test = "yoeo.test:run"
+yoeo-to-onnx = "scripts.convertPyTorchModelToONNX:run"
```

Comment:
Now this is located in the scripts package and not the YOEO python package.
This means that when installing YOEO as a python package outside a venv, the scripts might be missing or in a new `scripts` package which would be not so clean.

Reply:
We should think about reworking this structure